### PR TITLE
Custom TextField changes

### PR DESCRIPTION
**Pull request by anissa-agahchen**
Creating a customTextField to address tech debt

### CODE REVIEW

**Comment by jeznorth** [comment]
Looks good to me.

File: app/src/components/fields/CustomTextField.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  return (
+    <TextField
+      name={name}
+      label={label}
+      id={name}
+      onChange={handleChange}
+      variant="outlined"
+      value={values[name]}
+      fullWidth={true}
```
Comment (sdevalapurkar) [inline app/src/components/fields/CustomTextField.tsx:24]: Do we not have any textfields without full width? Shouldn't this be part of the other prop so we can change it if needed?

File: app/src/features/observations/components/BlockObservationForm.tsx [hunk 2]
```diff
@@ -194,33 +195,22 @@ const BlockObservationForm: React.FC<IBlockObservationFormProps> = (props) => {
...
-                  size="small"
+                <CustomTextField
                   name="block_name"
-                  label="Block ID"
-                  variant="outlined"
-                  value={values.block_name}
-                  onChange={handleChange}
-                  error={touched.block_name && Boolean(errors.block_name)}
-                  helperText={touched.block_name && errors.block_name}
+                  label={`Block ID`}
```
Comment (sdevalapurkar) [inline app/src/features/observations/components/BlockObservationForm.tsx]: this can just be:
```suggestion
                  label="Block ID"
```

**Review by sdevalapurkar — DISMISSED** [review]

File: app/src/components/fields/CustomTextField.tsx (hunk 1 above)
Comment (sdevalapurkar) [reply app/src/components/fields/CustomTextField.tsx:24]: lets leave it for now :)

File: app/src/components/fields/CustomTextField.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      error={touched[props.name] && Boolean(errors[props.name])}
```
Comment (sdevalapurkar) [inline app/src/components/fields/CustomTextField.tsx]:
```suggestion
      error={touched[name] && Boolean(errors[name])}
```

File: app/src/components/fields/CustomTextField.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+      helperText={touched[props.name] && errors[props.name]}
```
Comment (sdevalapurkar) [inline app/src/components/fields/CustomTextField.tsx]:
```suggestion
      helperText={touched[name] && errors[name]}
```

**Review by sdevalapurkar — APPROVED** [review]
noice

**Pull request merged**